modules/lang-mustache/src/test/java/org/elasticsearch/script/mustache/TemplateQueryBuilderTests.java

```diff
@@ -129,9 +129,9 @@ public void testUnknownField() throws IOException {
         String queryAsString = testQueryAsString.replace("inline", "bogusField");
         try {
             parseQuery(queryAsString);
-            fail("ScriptParseException expected.");
-        } catch (ParsingException e) {
-            assertTrue(e.getMessage().contains("unexpected field [bogusField], expected [inline, stored, file, lang, params]"));
+            fail("IllegalArgumentException expected");
```

Comment:
I might convert this to `expectThrows` while I'm there. I might not, depending on how much energy that takes.